tcms/testruns/tests/test_permissions.py

```diff
@@ -128,3 +128,31 @@ def verify_post_with_permission(self):
             self.assertEqual(case.history.latest().history_id, case_run.case_text_version)
             self.assertEqual(last_run.build, case_run.build)
             self.assertEqual(None, case_run.close_date)
+
+
+class GetTestRunViewTestCase(PermissionsTestCase):
```

Comment:
This is a new class of access level test case, one for which I don't have a naming convention.

After the main page is shown it tries to verify whether or not the bulk menu item "Add Comment" is present.